src/errors.js

```diff
@@ -11,6 +11,10 @@
 
 import type { BabelNodeSourceLocation } from "babel-types";
 
+// Information: Just an informative message with no semantic implications whatsoever
+// Warning: We came across something that has a well-defined meaning, but is likely to be wrong anyway.
+// RecoverableError: Something is clearly wrong and we'd like you to know, but, by sprinkling in some well-meaning assumptions, we could carry on the analysis to possibly give you even more diagnostics.
```

Comment:
Prepack might produce code that deviates in behavior from the original code, if the original code is not well behaved.